src/webportal/src/app/job-submission/components/docker-section.jsx

```diff
@@ -187,14 +187,14 @@ export const DockerSection = ({onValueChange, value}) => {
     if (optionKey !== 'customize-image' && isCutomizedImageEnabled) {
       setCutomizedImageEnabled(false);
     }
-  }, []);
+  }, [uri]);
```

Comment:
I think the `isCustomizedImageEnabled` state should be determined by 'uri' props.
You can use a `useMemo` hook to save the value.

Reply:
`isCustomizedImageEnabled ` is not just depends on the `uri` pop. Customers can also change its' value. So this situation could happens: `uri` not changed but `isCustomizedImageEnabled ` is changed by customer.
Maybe use state is better for this

Reply:
`useEffect` will be triggered each time page is rerendered. So if user input an existing docker option in the custom docker url text field like `ufoym/deepo:tensorflow-py36-cu100`, this hook will be triggered and `isCutomizedImageEnabled` will be set to `false`.